YellowNotes.Api/Program.cs

```diff
@@ -12,9 +17,20 @@ public static void Main(string[] args)
 
         public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
+                .ConfigureAppConfiguration((ctx, builder) =>
+                {
+                    var tokenProvider = new AzureServiceTokenProvider();
```

Comment:
you have to decide - ether configure things here or in `Startup.cs`

Reply:
Is it even possible to configure in `Startup`?  This is the Azure Key Vault configuration provider.

Reply:
you can when you take responsibility to build configuration with `ConfigurationBuilder`. never mind, let it be like it is :)